scadalts-ui/src/store/gv/storeIsAlive.js

```diff
@@ -0,0 +1,99 @@
+
```

Comment:
Is it possible o change name "t" as parameter in this file to more clear name - more understandable?